tests/conftest.py

```diff
@@ -577,3 +578,102 @@ def create_gcs_bucket(bucket_name: str):
 
     gcs_store = GcsStore(name=bucket_name, source="")
     return gcs_store
+
+
+# ----------------- Docker -----------------
+
+
+def run_shell_command_direct(subprocess, cmd: str):
+    # Run the command and wait for it to complete
+    result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
+
+    if result.returncode != 0:
+        print(result.stdout)
+
+    # Check for success
+    assert result.returncode == 0
+
+
+def run_shell_command(subprocess, cmd: list[str]):
+    # Run the command and wait for it to complete
+    result = subprocess.run(cmd, capture_output=True, text=True)
+
+    if result.returncode != 0:
+        print(result.stdout)
```

Comment:
for more visibility if there's a failure maybe print stderr instead or in addition?

Reply:
Indeed, good idea, thanks!
Done.